lib/components/fullpage.js

```diff
@@ -209,7 +210,7 @@ function getSlideCount(children) {
       return result;
     }
 
-    if (c.type.name === 'Slide') {
+    if (c.type === Slide) {
```

Comment:
so this is the only part that doesn't make sense to me. Before being passed through UglifyJS, c.type will equal Function